README.md

```diff
@@ -1,19 +1,19 @@
 # Introduction
-This repository is supposed to be used as a showcase project to show the usage of a number of feature for the [Neodymium Library](https://github.com/Xceptance/neodymium-library). It was created based on the [Neodymium template](https://github.com/Xceptance/neodymium-template).
+This repository is a showcase project to demonstrate the usage of several features of the [Neodymium Library](https://github.com/Xceptance/neodymium-library). It was created based on the [Neodymium template](https://github.com/Xceptance/neodymium-template).
```

Comment:
Is it really important that the Neodymium template was used to create this project?